apps/allocations/app/components/Panel/NewAllocation.js

```diff
@@ -13,7 +13,6 @@ import {
 } from '../Form'
 
 // TODO: Extract to shared
-const AVAILABLE_TOKENS = [ 'ETH', 'ANT', 'GIV', 'FTL', '🦄' ]
 const ALLOCATION_TYPES = [ 'Informational', 'Token Transfer' ]
```

Comment:
Weren't we supposed to remove informational?

Reply:
It's removed from the backend. Can we just remove the dropdown that selects between token transfer and Informational?